### DISCUSSION TITLE
Wallpapers random/slideshow

### DISCUSSION THREAD

**Discussion by petersen77** [helloSystem/hello#365] [Ideas]
I'm missing a function for randomize wallpapers and additional a slideshow. Which application is used for wallpaper selection?

**Comment by probonopd** [comment]
Right-click on the desktop, Desktop Settings. It's integrated in Filer.

**Comment by petersen77** [comment]
I can only set the Wallpaper & stretch it or fit to screen:

![Walppapers](https://user-images.githubusercontent.com/50071072/172234432-0de7a8f7-48b1-4061-b0f0-13a6ee15e30a.png)

**Reply by probonopd** [reply]
Yes, anything else is not implemented at the moment.
